node/src/effect.rs

```diff
@@ -967,6 +969,9 @@ impl<REv> EffectBuilder<REv> {
     }
 
     /// Asynchronously returns any missing descendant trie keys given an ancestor.
+    // TODO: get fast sync to not store trie nodes returned from storage and then remove this
+    // allow(unused)
+    #[allow(unused)]
```

Comment:
This seems to already be used by the fast sync code in operations:447. Is this attribute obsolete?